include/irr/asset/ICPUMeshBuffer.h

```diff
@@ -84,6 +84,14 @@ class ICPUMeshBuffer : public IMeshBuffer<ICPUBuffer, ICPUDescriptorSet, ICPURen
     {
         return &m_indexBufferBinding;
     }
+	inline void setIndexBufferBinding(asset::ICPUBuffer& buffer)
```

Comment:
`buffer` must be pointer

Reply:
also this function doesnt let setting offset in binding, it'd be better for it to take SBufferBinding&&

Reply:
resolved

Reply:
only have one function `void setIndexBufferBinding(SBufferBinding&& buffer)` also shouldn't the `IGPUMeshBuffer` have this function as well?